src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -57,6 +57,146 @@ public void TestInterpolatedVerbatimString()
             EOF();
         }
 
+        [Fact]
+        public void TestInterpolatedSingleLineRawString1()
```

Comment:
Consider adding a similar test for an error case like `@@@@` or `@@$$`, showing that it's still assumed to be a string literal (with missing quotes).

Reply:
Will do.

Reply:
we actually have tests for these AtThenDollar5 and MultipleAtSigns5

Reply:
I meant a test that shows the syntax tree (what kinds of syntax nodes are produced), not just the diagnostics